pages/projects/[slug].js

```diff
@@ -0,0 +1,17 @@
+import ProjectDetail from "../../components/ProjectDetail/ProjectDetail";
+import { useRouter } from "next/router";
+
+export default function ProjectDetailsPage({ projects }) {
+  const router = useRouter();
+  const { slug } = router.query;
+  if (!slug) {
```

Comment:
You can write your if condition e.g. like this:
```suggestion
if (!currentProject) {  
    return <div>Project not found</div>;
  }
```